src/sidebar-overlay/CommentBox/components/CommentBox.js

```diff
@@ -241,30 +250,32 @@ class CommentBox extends React.PureComponent {
                         [styles.noDisplay]: this.isHidden(),
                     })}
                 >
-                    <textarea
-                        rows={this.props.textareaRows}
-                        className={styles.textarea}
-                        value={this.props.commentInput}
-                        placeholder={'Add your comment...'}
-                        onChange={this.handleChange}
-                        onKeyDown={this.handleKeyDown}
-                        ref={this.setInputRef}
-                        onClick={() => this.props.setTagInput(false)}
-                    />
-                    <br />
-                    <div ref={this.setTagRef}>{this.renderTagInput()}</div>
-                    <div className={styles.buttonHolder}>
-                        <button
-                            className={styles.save}
-                            ref={this.setSaveRef}
-                            onClick={this.save}
-                        >
-                            Save
-                        </button>
-                        <a className={styles.cancel} onClick={this.cancel}>
-                            Cancel
-                        </a>
-                    </div>
+                    <form onSubmit={this.handleSubmit}>
```

Comment:
Do you think it's fine just to write `onSubmit={this.save}`, given that `handleSubmit` currently doesn't do anything apart from call `save`?

Reply:
Absolutely. Done.